modal/runner/containers/vllm_unified.py

```diff
@@ -24,7 +25,7 @@ def _make_container(
     class _VllmContainer(VllmEngine):
         def __init__(
             self,
-            model_path: str,
+            model_path: Path,
```

Comment:
Interesting - so it's possible to use `Path` as a param such that only 1 instance will be spawned to serve a certain path until the max concurrent input is reached?

Reply:
Yeah, I don't think change the type here should make a difference--looks like the part that controls that is recreating this class